### PR TITLE
Switch the on-push pipeline to GPU

### PR DESCRIPTION
**Pull request by syedriko**
Switch the on-push pipeline to GPU

### CODE REVIEW

**Comment by syedriko** [comment]
/approve
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-rag-content"

**Comment by syedriko** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-rag-content"

File: .tekton/lightspeed-rag-content-push.yaml [hunk 1]
```diff
@@ -243,11 +243,15 @@ spec:
         params:
         - name: name
           value: buildah-remote
-        - name: bundle
-          value: quay.io/konflux-ci/tekton-catalog/task-buildah-remote:0.2@sha256:eb3440a0ea2a1366160d86afdd17e04006aeba58e5b9ef95fbbcb9783cc59a68
+        - name: url
+          value: https://github.com/syedriko/build-definitions
```
Comment (xrajesh) [inline .tekton/lightspeed-rag-content-push.yaml:247]: Still pointing to personal fork @syedriko

File: .tekton/lightspeed-rag-content-push.yaml [hunk 2]
```diff
@@ -243,11 +243,15 @@ spec:
...
+        - name: revision
+          value: syedriko-gpu
```
Comment (xrajesh) [inline .tekton/lightspeed-rag-content-push.yaml:249]: @syedriko should name be syedriko-gpu ?

File: .tekton/lightspeed-rag-content-push.yaml (hunk 1 above)
Comment (syedriko) [reply .tekton/lightspeed-rag-content-push.yaml:247]: it's intentional. I need to merge this PR first to get .tekton/buildah-remote.yaml visible on the main branch and then in another PR I'll fix this to point to that file.

File: .tekton/lightspeed-rag-content-push.yaml (hunk 2 above)
Comment (syedriko) [reply .tekton/lightspeed-rag-content-push.yaml:249]: same as above

**Pull request merged**